File: app/src/main/java/app/country/information/model/Car.java

package app.country.information.model;

import java.util.List;
import javax.annotation.Generated;
import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

@Generated("jsonschema2pojo")
public class Car {

    @SerializedName("signs")
    @Expose
    public List<String> signs = null;
    @SerializedName("side")
    @Expose
    public String side;

}

File: app/src/main/java/app/country/information/model/Car.kt
package app.country.information.model

import com.google.gson.annotations.SerializedName
import com.google.gson.annotations.Expose

class Car {
    @SerializedName("signs")
    @Expose
    var signs: List<String>? = null

    @SerializedName("side")
    @Expose
    var side: String? = null
}